draft-ietf-scone-protocol.md

```diff
@@ -260,8 +260,8 @@ from {{Section 4 of INVARIANTS}}.
 SCONE Packet {
   Header Form (1) = 1,
   Reserved (1),
-  Rate Signal (6),
-  Version (32) = 0x6f7dc0fd or 0xef7dc0fd,
+  Rate Signal (7),
+  Common version bits (31) = 0x6f7dc0fd,
```

Comment:
+1 to @martinthomson that the definition of these fields should better be consistent with that of RFC 8999 and 9000. At the same time, +1 to @ihlar to describe the signal as a 7-bit value, rather than a combination of a 6-bit field and the two QUIC versions.

Therefore, maybe change the field definitions to something like below, and explain that the Rate Signal is conveyed as a 7-bit value spanning across the two consecutive fields? Then, in the rest of the document, we can just talk about the "Rate Signal."

```suggestion
  Lower 6 Bits (6),
  Version (32) = 0x6f7dc0fd or 0xef7dc0fd,
```

Reply:
```suggestion
  Rate Signal High Bits (6),
  Version (32) = 0x6f7dc0fd or 0xef7dc0fd,
```

Then we can say a Rate Signal comprises "Rate Signal High Bits" and an LSB which is the MSB of the "Version" field.

Reply:
Yes. That's exactly what I suggested in my previous comment.